edb/schema/database.py

```diff
@@ -39,7 +42,24 @@ class DatabaseCommandContext(sd.ObjectCommandContext):
 
 class DatabaseCommand(sd.GlobalObjectCommand, schema_metaclass=Database,
                       context_class=DatabaseCommandContext):
-    pass
+
+    def _create_begin(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext,
+    ) -> s_schema.Schema:
+        schema = super()._create_begin(schema, context)
+
+        # Validate that the database name is fewer than 64 characters
+        name = self.get_attribute_value('name')
+        if len(str(name)) > 63:
```

Comment:
Please use a constant instead of a magic number.  We have `MAX_NAME_LENGTH` in `pointers.py`, factor it out somewhere.

Reply:
`server/defines.py` would be an OK place since we have so many other constants there.

Reply:
Yes, although I'd move it out or we'll be importing `server` everywhere.

Reply:
I've factored out the 63 constant into `server/defines.py` from `database.py`, `pointers.py` and `common.py`.